lib/YuiRestClient.pm

```diff
@@ -135,4 +135,14 @@ sub set_libyui_backend_vars {
     set_var('YUI_SERVER', $server);
 }
 
+sub setup_libyui_firstboot {
+    my $port = get_var('YUI_PORT');
```

Comment:
> I would suggest to remove module 'check_libyui_firstboot.pm' and move `YuiRestClient::connect_to_app()` to 'yast2_firstboot' module.

I prefer this way, because if you see the test failing, you can pause it, check the "details" tab to see what port is used and connect quickly with VNC. Otherwise, you don't know what port is used (though, you could deduct it) as the module did not pass yet. It cannot be checked earlier as libyui is not yet active before reboot.

Reply:
you can still display both [here](http://waaa-amazing.suse.cz/tests/14988#step/setup_libyui_firstboot/7) (which is before than where is set now) and the connection just before acting on the UI. Am I missing something?

Reply:
Well we would need to start a module to start libyui at that point no ? We could just show the port, but we would miss the info if libyui is broken after reboot, so it would be harder to troubleshoot in such case.

Reply:
Ah, sorry, now I understand what you mean

Reply:
So I applied what you suggested.